features/step_definitions/matching.rb

```diff
@@ -0,0 +1,16 @@
+Given(/^that the algorithm matches me with "(.*?)", "(.*?)", "(.*?)", "(.*?)", and "(.*?)"$/) do |match1, match2, match3, match4, match5|
+  pending
```

Comment:
@smartfuse can you tell me how I should force `matches` to return a preset list of users?